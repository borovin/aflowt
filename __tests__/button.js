const Browser = require('../tools/browser');

test('Primary button example', async () => {
    const selector = 'b-button[color="primary"]';
    const browser = new Browser();

    browser.goto('/examples/button.html');

    const snapshot = await browser.snapshot(selector);

    expect(snapshot).toMatchSnapshot();

    browser.close();
});

test('Accent button example', async () => {
    const selector = 'b-button[color="accent"]';
    const browser = new Browser();

    browser.goto('/examples/button.html');

    const snapshot = await browser.snapshot(selector);

    expect(snapshot).toMatchSnapshot();

    browser.close();
});