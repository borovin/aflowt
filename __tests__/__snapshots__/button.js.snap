// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accent button example 1`] = `
"<b-button color=\\"accent\\">
    <button>
        <slot>Accent button</slot>
    </button>
</b-button>"
`;

exports[`Primary button example 1`] = `
"<b-button color=\\"primary\\">
    <button>
        <slot>Primary</slot>
    </button>
</b-button>"
`;

exports[`Primary flat button example 1`] = `
"<b-button flat=\\"\\" color=\\"primary\\">
    <button>
        <slot>Primary flat</slot>
    </button>
</b-button>"
`;
